regexes/device/mobiles.yml

```diff
@@ -2869,14 +2874,16 @@ Casio:
 
 # Casper
 Casper:
-  regex: 'CASPER_|VIA_(A3|F[12]|G4|M4|L8|P2|V8C|A1_Plus|A1|S10|S)|VIA-T7D[);/ ]'
+  regex: 'CASPER_|VIA[- _](T7D|A3|F[12]|G4|M4|L8|P2|V8C|A1_Plus|A1|S10|S|T17_M|T17)[);/ ]'
   device: 'smartphone'
   models:
     - regex: 'CASPER_VIA_(G1|E1|[AE]2|M[123]|V[35689]|V10)[);/ ]'
       model: 'VIA $1'
     - regex: 'CASPER_VIA_E1c'
       model: 'VIA E1C'
-    - regex: 'VIA[- _](T7D|S10|L8)[);/ ]'
+    - regex: 'VIA_(A3|F[12]|G4|M4|P2|V8C|A1_Plus|A1)'
```

Comment:
Isn't that line almost the same as a few lines below? guess could be simply removed again 🤔

Reply:
done